website/install.rst

```diff
@@ -39,15 +39,12 @@ On macOS, install via `Homebrew <https://formulae.brew.sh/formula/openexr>`_:
 
    % brew install openexr
 
-We do not recommend installation via
-`Macports <https://ports.macports.org/port/openexr>`_ because the
-distribution is out of date.
-
-Also note that the official OpenEXR project does not provide supported
-python bindings. ``pip install openexr`` installs the `openexrpython
-<https://github.com/jamesbowman/openexrpython>`_ module, which is not
-affiliated with the OpenEXR project or the ASWF. Please direct
-questions there.
+Alternatively, you can install on macOS via `MacPorts
+<https://ports.macports.org/port/openexr>`_:
+
+.. code-block::
+
+   % sudo port install openexr
```

Comment:
is sudo needed here?

Reply:
I don't think we install anything into a protected area.

Reply:
I lifted that from https://ports.macports.org/port/openexr, but yes, probably better to omit it in our instructions. At the very least it should be consistent with the brew instructions. 

I also added a reference to vcpkg for Windows.